### ISSUE TITLE
put bitmap directly into cache without having ImageView

### ISSUE THREAD

**Issue by shankar667** [DWorkS/VolleyPlus#57]
Can you help me out, how can i put the custom bitmap into cache with key without having other parameters like ImageView, etc..?

**Comment by Emasoft** [comment]
Yes, this is an important issue for me too. I want to keep the code for downloading the images separated and independent from the code to display it.
